app/views/admin/site_customization/pages/edit.html.erb

```diff
@@ -1,11 +1,11 @@
-<% provide :title do %>
-  <%= t("admin.header.title") %> - <%= t("admin.menu.site_customization.pages") %> - <%= @page.title %>
+<% provide :slug do %>
+  <%= t("admin.header.title") %> - <%= t("admin.menu.site_customization.pages") %> - <%= @page.slug %>
```

Comment:
I don't have a strong preference about which title the browser should show, which is what we're changing here. Which one do you think it's better?

Reply:
I don´t have preference either 😉. Just confirm me what to use. :)